_layouts/documentation.html

```diff
@@ -24,4 +24,17 @@ <h1 class="documentation-title">{{ page.title }}</h1>
     <div class="documentation-content">{{ content }}</div>
   </article>
 </div>
+<div id='discourse-comments'></div>
+
+<script type="text/javascript">
+ record_from_txt = document.getElementsByClassName('documentation-source')[0].getElementsByTagName("a")[0].href
+  DiscourseEmbed = { discourseUrl: 'http://meta.discourse.org/',
+                     topicId: parseInt(txt.substring(txt.lastIndexOf("/")+1)) };
```

Comment:
this may fail since we also have some docs from GitHub. Better not to invoke the function below.